code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -484,6 +435,7 @@
 	category = "Rituals"
 	refundable = 0
 	buy_word = "Cast"
+	cost = 2

```

Comment:
This is pointless. The parent spellbook_entry has the cost set to 2. There is no reason to set it again.